### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by DrJJWMac** [latex3/pdfresources#32]
I have been in desperate search for a way to generate a basic PDF/A compliant document through LaTeX. My frustrations were in part in ignorance about the meanings for the standards and in part from ongoing confusion about why various approaches would or would not work for me. I was bouncing between documents generated with hyperxmp and pdfx, and between trials on macOS with Adobe Acrobat, Preview, and PDFExpert.

I've now tested a MWE using the \RequirePackage{pdfmanagement-testphase} and \DocumentMetadata{...} headers with \hyperxmp (my preference for its ease using syntax directly in \hypersetup{...}). The combination immediately fixed what was an annoying and ongoing issue: some key meta-data (subject, keywords) would only appear in the Acrobat document properties but not in the document properties viewed via other macOS specific PDF apps (e.g. Preview or PDFExpert). If I understand the details correctly, the use of the pdfmanagement package seems also to assure that packages such as hyperxmp or pdfx store meta-content such as author, title, subject, and keywords both in the XMP fields as well as in the INFO fields.

In summary, I can now generate a PDF/A-2u document that passes the compliant checks at that level and that has the four key meta fields (title, author, keywords, and subject) appearing as document properties across all of my PDF editor apps on macOS.

So ... many thanks! I much appreciate the developments being done here to standardize (and indeed now simplify) how PDF/A level documents can be generated from *TeX source files.

**Comment by u-fischer** [comment]
Thanks for the feedback. Just a comment: The bundle contains patches for hyperxmp and if there are problems we will try to resolve them. But the pdfx package is not compatible, so don't use it together with the new pdfmanagement.
